sensors/mpu6050/README.md

```diff
@@ -0,0 +1,19 @@
+
+# Connections
+
+- No wires are necessary, simply connect the MPU6050 to the corresponding header on the board
```

Comment:
## Imperative Language in Docs

Connect the MPU6050 to the correspond header on the board directly.